jenkins/scripts/bml_integration_test.sh

```diff
@@ -56,6 +55,7 @@ scp \
   -r \
   "${CI_DIR}/files/run_integration_tests.sh" \
   "${CI_DIR}/files/vars.sh" \
+  "${CI_DIR}/files/integration_test_env.sh" \
```

Comment:
I really wanted to reuse `vars.sh` but I couldn't find a good solution. The issue is that in the other pipelines everything is set up through `run_integration_tests.sh` so it is enough to get the variables in the environment used there, but for BML we use the `deploy-lab.yaml` playbook so we need to have the same environment available there. Therefore I decided to use the `integration_test_env.sh` as a `config_{user}.sh` file with BML.